packages/app/src/cli/models/extensions/specifications/flow_trigger.ts

```diff
@@ -1,37 +1,17 @@
-import {BaseSchema} from '../schemas.js'
+import {BaseSchemaWithHandle} from '../schemas.js'
 import {createExtensionSpecification} from '../specification.js'
 import {validateNonCommerceObjectShape} from '../../../services/flow/validation.js'
 import {serializeFields} from '../../../services/flow/serialize-fields.js'
 import {zod} from '@shopify/cli-kit/node/schema'
 
-export const FlowTriggerExtensionSchema = BaseSchema.extend({
-  name: zod.string(),
-  description: zod.string().optional(),
+export const FlowTriggerExtensionSchema = BaseSchemaWithHandle.extend({
   type: zod.literal('flow_trigger'),
-  extensions: zod
-    .array(
-      zod.object({
-        type: zod.literal('flow_trigger'),
-        schema: zod.string().optional(),
-        return_type_ref: zod.string().optional(),
-      }),
-    )
-    .min(1),
-  settings: zod
-    .object({
-      fields: zod
-        .array(
-          zod
-            .object({
-              key: zod.string().optional(),
-              description: zod.string().optional(),
-              type: zod.string(),
-            })
-            .refine((field) => validateNonCommerceObjectShape(field, 'flow_trigger')),
-        )
-        .optional(),
-    })
-    .optional(),
+  schema: zod.string().optional(),
```

Comment:
FYI we're removing the `schema` and `return_type_ref`  fields since we won't support them for triggers for Editions. Here's @mikee649's [PR](https://github.com/Shopify/cli/pull/2401) that handles that.

Reply:
i'll remove those in this PR as well